app/controllers/concerns/europeana/search_api_consumer.rb

```diff
@@ -39,6 +39,19 @@ def search_results_for_europeana_ids(europeana_ids, **options)
       end
     end
 
+    # Queries the Search API for Europeana records having another as part of them
+    #
+    # @param europeana_id [String] Europeana record ID to search for records
+    #   having is as part of them
+    # @return see Europeana::API::Record.search
+    def search_results_for_dcterms_has_part(europeana_id)
+      search_options = {
+        api_url: api_url, rows: blacklight_config.default_per_page,
```

Comment:
this will retrieve 12 records, if we're only interested in 1 parent item perhaps this method should take per_page as an option. Then it can still default to this.